conn.go

```diff
@@ -141,8 +141,8 @@ type ConnConfig struct {
 	ConnectTimeout  time.Duration
 	ReadTimeout     time.Duration
 	WriteTimeout    time.Duration
-	ProtoVersion    int
 	Keepalive       time.Duration
+	ProtoVersion    int
```

Comment:
My guess that it considers 32 bit architectures. there is no other cases when it would assume that int could be less in size than `time.Duration`.
But driver won't even build for 32 arch, so these changes do not make any sense.